### DISCUSSION TITLE
Missing files in submodule

### DISCUSSION THREAD

**Discussion by lharrill01** [hathach/tinyusb#915] [Q&A] [answered]
I am getting these error when running the tinyusb submodule on windows 10. I am new to using submodules and I need some guidance on rectifying these issues.

![image](https://user-images.githubusercontent.com/6504801/123128857-7b218f80-d419-11eb-81a2-de6f144c62e5.png)

are these files just missing in the submodule or should they be removed?

**Comment by hathach** [comment, marked as answer]
check out the getting started here https://github.com/hathach/tinyusb/blob/master/docs/getting_started.md
